presto-router/pom.xml

```diff
@@ -162,17 +169,21 @@
             <artifactId>jmxutils</artifactId>
         </dependency>
 
-        <!-- test dependencies-->
         <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-            <scope>test</scope>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-main</artifactId>
         </dependency>
-
+        <!-- test dependencies-->
         <dependency>
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-main</artifactId>
             <scope>test</scope>
+            <type>test-jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <scope>test</scope>
```

Comment:
Let's try to maintain the ordering of the dependencies to minimize the diff. Please put this one below the test section comment